example/.flutter-plugins-dependencies

```diff
@@ -1 +1 @@
-{"info":"This is a generated file; do not edit or check into version control.","plugins":{"ios":[{"name":"testfairy","path":"/Users/diego/Dropbox/Projects/testfairy/testfairy-flutter/","dependencies":[]}],"android":[{"name":"testfairy","path":"/Users/diego/Dropbox/Projects/testfairy/testfairy-flutter/","dependencies":[]}],"macos":[],"linux":[],"windows":[],"web":[]},"dependencyGraph":[{"name":"testfairy","dependencies":[]}],"date_created":"2020-08-05 17:13:22.906563","version":"1.17.5"}
\ No newline at end of file
+{"info":"This is a generated file; do not edit or check into version control.","plugins":{"ios":[{"name":"testfairy","path":"/Users/diego/Dropbox/Projects/testfairy/testfairy-flutter/","dependencies":[]}],"android":[{"name":"testfairy","path":"/Users/diego/Dropbox/Projects/testfairy/testfairy-flutter/","dependencies":[]}],"macos":[],"linux":[],"windows":[],"web":[]},"dependencyGraph":[{"name":"testfairy","dependencies":[]}],"date_created":"2020-08-11 13:32:10.891452","version":"1.17.5"}
```

Comment:
@diegoperini do we need this file in git?

Reply:
Good catch, it's garbage. 👍